storager_test.go

```diff
@@ -488,3 +489,57 @@ func TestStorage_Fetch(t *testing.T) {
 		assert.Error(t, err)
 	}
 }
+
+func TestStorage_Create(t *testing.T) {
```

Comment:
I found mock is not a suitable test framework for us.

Maybe we can introduce bdd tests here to replace them.

Reply:
This can be implement in another PR,